src/containers/OrgDetails.tsx

```diff
@@ -0,0 +1,252 @@
+        const vatParsed = eeRegExp.exec(values['vat']) as EeRegExp | null;
+        if (vatParsed?.groups.NUM) {
+          isVatValid(vatParsed.groups.NUM)
+            .then(setVatValid)
+            // eslint-disable-next-line no-console
+            .catch(console.error);
+        }
+      }
+    });
+    return () => subscription.unsubscribe();
+  }, [watch]);
+
+  const onSubmit = useCallback(
+    async (values: OrganizerInfo) => {
+      logger.info('Submit organizer data', values);
+      try {
+        setError(undefined);
+
+        // navigate('/checkout/' + ...);
+      } catch (error) {
+        const message = (error as Error).message || 'Unknown error';
+        setError(message);
+      }
+    },
+    [groupCheckout]
+  );
+
+  return (
+    <FormProvider methods={methods} onSubmit={handleSubmit(onSubmit)}>
+      {error && <Alert severity="error">{error}</Alert>}
+      <Card sx={{ my: 3, p: 3 }}>
+        <Typography variant="h3">Get a quotation for your stay</Typography>
+        <Typography sx={{ mb: 3 }}>
+          This Information will be strictly used by the hotel and win.so to confirm the
+          booking issue the invoice.
+        </Typography>
+        <Box
+          sx={{
+            display: 'grid',
+            columnGap: 2,
+            rowGap: 3,
+            gridTemplateColumns: { xs: 'repeat(1, 1fr)', sm: 'repeat(1, 1fr)' }
+          }}
+        >
+          <RHFTextField name="firstname" label="First Name *" />
+          <RHFTextField name="lastname" label="Last Name *" />
+          <RHFPhoneField name="phone" label="Phone Number *" />
+          <RHFTextField name="email" label="Email Address *" />
+          <RHFTextField name="companyName" label="Company Name" />
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: 'row',
+              alignItems: 'flex-start',
+              gap: 1
+            }}
+          >
+            <RHFAutocomplete
+              name="country"
+              label="Country (Optional)"
+              options={countriesOptions}
+            />
+            <RHFTextField name="city" label="City (Optional)" />
+          </Box>
+          <RHFTextField name="streetName" label="Street name and number (Optional)" />
+          <RHFTextField
+            name="vat"
+            label="VAT Number (optional)"
+            InputProps={{
+              // Show green checkmark if VAT is valid only
+              endAdornment: vatValid ? (
+                <Iconify
+                  color="green"
+                  icon="akar-icons:circle-check-fill"
+                  marginLeft={1}
+                />
+              ) : null
+            }}
+          />
+          <RHFCheckbox
+            sx={{
+              pl: 2
+            }}
+            name="invoiceRequired"
+            label={<Typography variant="subtitle1">I will need an invoice</Typography>}
+          />
+
+          <FormControlLabel
```

Comment:
is there a reason why we won't use the `RHFCheckbox` for the terms and conditions?

Reply:
this code also has been copied from the UserInfo form. I think this is because the `privacy` variable is not a part of the form state.